extensions/cloud/portability-cloud-microsoft/src/main/java/org/dataportabilityproject/cloud/microsoft/cosmos/DataWrapper.java

```diff
@@ -0,0 +1,35 @@
+package org.dataportabilityproject.cloud.microsoft.cosmos;
+
+import com.microsoft.azure.storage.table.TableServiceEntity;
+
+/** Wraps Job data for serialization to Cosmos DB */
```

Comment:
Maybe a link to the api docs for the specific requirements of this class. Maybe we want to call it CosmoDataWrapper?